### PR TITLE
[MOD] Easy CBMs

### PR DESCRIPTION
**Pull request by Fris0uman**
#### Summary
```SUMMARY: Mods "CBMs from zombies are clean, manual install is enabled, safe autodoc is enabled."```

#### Purpose of change
CBMs harvested from zombies are ready to be installed.
Safe autodoc and Manual install are merged in this mod
Requires #32013 

#### Describe the solution
Removes flags from harvest entries

#### Additional context
CBMs acquired through surgery will still be filthy, faulty and non sterile.

### CODE REVIEW

File: data/mods/No_Filthy_CBMs/modinfo.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "type": "MOD_INFO",
+    "ident": "no_filthy_cbm",
+    "name": "No Filthy CBM",
```
Comment (Night-Pryanik) [inline data/mods/No_Filthy_CBMs/modinfo.json]: I guess this short name could be slightly misleading for some individuals, especially if they don't read the description.
```suggestion
    "name": "No Filthy CBMs From Zombies",
```

**Comment by Mecares** [comment]
Maybe it would be better to implement it into one of the other cbm easymode mods?

**Comment by kevingranade** [comment]
I'd much rather have one mod that works the way *someone* thinks is a good idea than all these tiny single-purpose mods cluttering up menus

**Comment by kasanryukin** [comment]
> I'd much rather have one mod that works the way _someone_ thinks is a good idea than all these tiny single-purpose mods cluttering up menus

I don't like the idea of all or nothing here for CBMs. I like safe autodoc, but I don't think manual install and clean/pre-packed cbms should be forced to be used at the same time.  Might as well debug install them at that point.

**Comment by Night-Pryanik** [comment]
I'd say manual install and safe autodoc are mutually exclusive mods: you don't care whether autodoc installation is safe or not if you're using manual install.

**Comment by Mecares** [comment]
I don't use those mods but I understood it as such: they are combined often to give a advantage to use the autodoc over manual installation e.g. giving the player the choice installing right now manually with higher risk or playing it save using an autodoc.

**Comment by Moonsabrt** [comment]
As the other people say 'w')
I like safe autodoc and the mod thing of CBMS being clean when harvested but dislike the portion where of manual install. Having that level of customization while it may clutter menus gives people the freedom to 'pick and choose' what they are comfortable with.

**Comment by Fris0uman** [comment]
oups didn't meant to do that

**Comment by NancokPS2** [comment]
Was this ever merged? the safe autodoc mod doesn't seem to work on my end so i was wondering if it got obsoleted in favor of being put in the main game(?)

**Comment by Night-Pryanik** [comment]
No, it wasn't merged.

**Comment by Fris0uman** [comment]
This mod doesn't fit the requirements to be part of the repo. But you can find it there https://github.com/Fris0uman/Easy-CBM-Mods

**Pull request closed without merging**